samples/f77/SConscript

```diff
@@ -34,7 +34,7 @@ mak_path = pjoin(localenv['ct_incroot'], 'cantera', 'Cantera.mak')
 if ' ' in mak_path:
     # There is no reasonable way to handle spaces in Makefile 'include'
     # statement, so we fall back to using the relative path instead
-    mak_path = os.path.relpath(mak_path, pjoin(localenv['ct_sampledir'], 'f90'))
+    mak_path = relpath(mak_path, pjoin(localenv['ct_sampledir'], 'f90'))
```

Comment:
I'm wondering if this should be `'f77` at the end of the line?

Reply:
I believe this is correct - it's in the `samples/f77` folder, after all. Please update and I can merge. The math constant issue that popped up in the same UG post is somewhat gnarlier, and very different from what is addressed here.

Reply:
@ischoegl Done! Used the new `github.dev` editor, it was pretty seamless and nice!